src/magic.c

```
@@ -313,7 +295,7 @@ void spell_sunburst(byte level, struct char_data *ch,
 	if (number(1, 15) == 1)
         spell_blindness(level, ch, victim, 0);
     // sprintf(buf, "DEBUG: sunburst: %d", dam);
-    log(buf);
+    mudlog(buf);
```

Cppcheck (2.18.0)

[error] Uninitialized variable

(uninitvar)